.github/workflows/spelling_action.yml

```diff
@@ -0,0 +1,15 @@
+name: Spellcheck Action
+on: push
+
+jobs:
+  build:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: rojopolis/spellcheck-github-actions@0.11.0
+      name: Spellcheck
+      with:
+        source_files: "**/*.md"
```

Comment:
I see you have added the source_files tab, but one important thing is missing.

See this [issue](https://github.com/rojopolis/spellcheck-github-actions/issues/33) I opened in the Repo, I have posted a solution I found as well along with other available solutions (which are better).

Kindly incorporate them in your action as well.

The thing is as of now this action will create a lot of noise because it will check each and every `md` file in the codebase whenever a file is pushed/pull request made. Obviously, we don't want that. The solution for the problems can be found in the thread I mentioned above.

Please try to integrate them here as well.